src/opensynth/models/faraday/gaussian_mixture/prepare_gmm_input.py

```diff
@@ -0,0 +1,33 @@
+import torch
+
+from opensynth.models.faraday.losses import _expand_samples
+from opensynth.models.faraday.model import FaradayVAE
+
+
+def prepare_data_for_model(
+    vae_module: FaradayVAE, data: torch.Tensor
+) -> torch.Tensor:
+    """Prepare data for the GMM by encoding it with the VAE.
+     If sample weights are used, then expand the repeat based on the weights
+      and shuffle the expanded dataset.
```

Comment:
```suggestion
    """Prepare data for the GMM by encoding it with the VAE.
     If sample weights are used, then expand the repeat based on the weights
     and shuffle the expanded dataset.
```

Just fixing indenting

Reply:
Updated to suggested changes.